storage/aptosdb/src/lib.rs

```diff
@@ -167,23 +171,18 @@ fn error_if_too_many_requested(num_requested: u64, max_allowed: u64) -> Result<(
 }
 
 fn error_if_version_is_pruned(
-    pruner: &Option<Pruner>,
-    pruner_index: PrunerIndex,
+    pruner: &(dyn PrunerManager + Send + Sync),
```

Comment:
Doesn't need to be `Send`? 

and nit: Shall we just make `trait PrunerManager: Sync` and avoid repeating it in places like this? like https://github.com/aptos-labs/aptos-core/blob/da86b2f29d67694713584e8b9b92f0b467af5f26/storage/state-view/src/lib.rs#L22

Reply:
Done